src/ComServerHelpers/Internal/Windows/Com/Marshalling/TrustLevelMarshaller.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.InteropServices.Marshalling;
+using Windows.Win32.System.WinRT;
+
+namespace ComServerHelpers.Internal.Windows.Com.Marshalling;
+
+[CustomMarshaller(typeof(TrustLevel), MarshalMode.UnmanagedToManagedOut, typeof(TrustLevelMarshaller))]
+[CustomMarshaller(typeof(TrustLevel), MarshalMode.UnmanagedToManagedIn, typeof(TrustLevelMarshaller))]
+[CustomMarshaller(typeof(TrustLevel), MarshalMode.ManagedToUnmanagedOut, typeof(TrustLevelMarshaller))]
+[CustomMarshaller(typeof(TrustLevel), MarshalMode.ManagedToUnmanagedIn, typeof(TrustLevelMarshaller))]
+internal static class TrustLevelMarshaller
```

Comment:
`TrustLevel` should be an enum, is this marshaller actually needed?

Reply:
If `TrustLevel` is defined in another assembly, technically yes.

Reply:
Right, but I meant this is just an internal enum defined by CsWin32, right? 🤔

Reply:
Yeah but if it's defined in a different assembly (I don't think so because it's CsWin32) the source generator can't guarantee that the underlying type isn't changed out from under it to something non-blittable.

If the code compiles without it, it shouldn't be added (as it's not required at that point).

Reply:
I will see if it compiles without it, I suppose it does.

Reply:
Removed.